codeception.yml

```diff
@@ -3,6 +3,8 @@ actor: Tester
 
 include:
     - tests/Common
+    - tests/Shared
+    - tests/Zed
```

Comment:
Are we switching to consistent yml 2 space rule in all repos?
Then https://github.com/spryker/architecture-sniffer/blob/master/.editorconfig#L18 is also a requriement for this code base :)